pages/common/magick.md

```diff
@@ -1,6 +1,7 @@
-# ImageMagick
+# magick
 
 > Create, edit, compose, or convert bitmap images.
+> ImageMagick version 7+. See `convert` for versions 6 and below.
```

Comment:
Perhaps this should be `tldr convert` or `convert tldr page`, since I guess . What do you think?

Reply:
I'm not sure what you're getting at. This isn't an example - it's a description line with a see also - e.g. like `arch`, `lsmod`, etc.

Reply:
Ah okay, nevermind. I understood what you meant now.